src/pytorch_lightning/strategies/ipu.py

```diff
@@ -41,28 +41,6 @@
     poptorch = None
 
 
-class LightningIPUModule(_LightningModuleWrapperBase):
```

Comment:
to deprecate the class, keep it and do:
```python
class LightningIPUModule(_LightningModuleWrapperBase):
    def __init__(self, ... same old args):
        rank_zero_deprecation('`LightningIPUModule` has been deprecated in v1.8.0 and will be removed in v2.0.0')
        ... same old code
```
and a test in `tests/tests_pytorch/deprecated_api/test_remove_2-0.py`.